src/main/java/appeng/me/GridNode.java

```diff
@@ -227,6 +230,8 @@ public void destroy() {
         if (this.myGrid != null) {
             this.myGrid.remove(this);
         }
+
+        destroyStatus = 2;
```

Comment:
You will never need this last state, so the `destroyStatus` can be boolean and called like `isDestrroyed` or better `isAlive`

Reply:
`isAlive` sounds good, I change it. :)